src/app/locales/ja/settings.yml

```diff
@@ -97,6 +97,23 @@ streaming:
         parameters:
             title: カスタムパラメータ
             placeholder: カスタムパラメータの追加
+    webbrowser:
+        title: StreamlinkのWebブラウザAPI
```

Comment:
Please keep the term "webbrowser API" untranslated because it's the name of the specific Streamlink interface. It's using the single word "webbrowser" for this very reason.

Reply:
```suggestion
        title: Streamlink webbrowser API
```